models/review.go

```diff
@@ -98,13 +104,28 @@ func (r *Review) LoadReviewer() error {
 	return r.loadReviewer(x)
 }
 
+// LoadRequestDoer load review request doer
+func (r *Review) LoadRequestDoer() error {
+	if r.Type == ReviewTypeRequest {
+		doer, err := GetUserByName(r.Content)
```

Comment:
Hello, I think it's not necessay to load RequestDoer full message , Because I  I just need ``doerID`` to determine user permissions in tmpl file and    `` isLegalReviewRequest``, other message are not used. I found a other item in Review type named ``OriginalAuthorID`` , but not in ``CreateReviewOptions``, can I reuse it to save ``doerID``? Thanks

Reply:
OriginalAuthorID is for issues/prs that were migrated from another service like Github. It stores the Github user id as OriginalAuthorID for the case that the user then creates an account on Gitea and authenticates with Github login then we can re-assign those migrated issues to their new Gitea account.

Reply:
You could simply add another column to the `Comment` table.